Packs/MicrosoftDefenderAdvancedThreatProtection/Playbooks/playbook-MDE_Malware_-_Incident_Enrichment.yml

```diff
@@ -1568,6 +1543,59 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "56":
+    id: "56"
+    taskid: 75a12361-2f13-4b7e-8179-917b1e74712a
+    type: condition
+    task:
+      id: 75a12361-2f13-4b7e-8179-917b1e74712a
+      version: -1
+      name: Check if Alert Info was Fetched to Incident Fields
+      description: Ensure that the device name  and alert name incident fields are not empty.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "16"
+      "yes":
+      - "46"
+      - "47"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              complex:
+                root: incident
```

Comment:
Just to be sure -> those always will be always met (assuming it as we are doing it through the mapper)
So I was thinking maybe we should check the evidence part instead 
No `Evidence` key = no need to populate that info in the alert tab. WDYT?

Reply:
Those will not always be met - sometimes the alert name or device name is empty cause the details are missing from the alert.
according to out MDE alerts most of the time the Evidence key is empty...

Reply:
changed the conditional task to check if the 'MicrosoftATP.Alert.Evidence' context key is not empty